src/application-navigation/terra-application-navigation/utility-menu/_UtilityMenu.jsx

```diff
@@ -58,8 +58,8 @@ const defaultProps = {
   utilityItems: [],
 };
 
-const utilityMenuSettingsKey = 'terra-application-navigation.utility-menu.settings';
-const utilityMenuHelpKey = 'terra-application-navigation.utility-menu.help';
+const utilityMenuSettingsKey = '@cerner/terra-application-navigation.utility-menu.settings';
+const utilityMenuHelpKey = '@cerner/terra-application-navigation.utility-menu.help';
```

Comment:
These keys probably don't need the scope

Reply:
lol, thats what you get when you find and replace